docker/.env

```
@@ -289,3 +289,10 @@ THREAD_POOL_MAX_WORKERS=128
 
 #Option to disable login form for SSO
 DISABLE_PASSWORD_LOGIN=false
+
+# OpenDataLoader PDF parser (standalone service — no JRE on the RAGFlow host).
+# Start the service with: docker compose --profile opendataloader up -d
+# Then uncomment and set OPENDATALOADER_APISERVER to the service URL:
+# OPENDATALOADER_APISERVER=http://opendataloader:9383
+
+MYSQL_PORT=3307
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The MYSQL_PORT key is duplicated

(DuplicatedKey)